src/test/testData.ts

```diff
@@ -74,9 +74,132 @@ const gifts: IGift[] = [
   }
 ]
 
+const realGifts: IGift[] = [
+  {
+    amount: 10,
+    cost: 100,
+    description: '',
+    name: 'Chocolate / Rafaello / Craft Beer :candy:',
+    teamId: ''
```

Comment:
Why we have teamId here when it's empty? If it's because of IGift try using Omit